dallinger/command_line/docker_ssh.py

```diff
@@ -277,7 +275,26 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def get_dotenv_values(executor):
+    dotenv_content = executor.run(
+        "test -f ~/dallinger/.env && cat ~/dallinger/.env", raise_=False
+    )
+    if dotenv_content:
+        return dotenv_values(stream=io.StringIO(dotenv_content))
```

Comment:
A naive approach could be to split each line on the leftmost `=`, and use what's on its left as variable name, and on what's on its right as variable value. This would allow us to not depend on `dotenv`.

Reply:
A .env file can include comments, quoted or unquoted variables and other edge cases.
As agreed via dm i changed the implementation to use a .env.json file, which will simplify code and doesn't require additional dependencies or specific code just to handle the syntax.